### DISCUSSION TITLE
Bad Documentation

### DISCUSSION THREAD

**Discussion by bwlove319** [theory/sqitch#611] [General]
Many of the sample commands in the documentation result in errors on the command line.
![image](https://user-images.githubusercontent.com/87549506/156466444-8d3cfb0d-d904-4294-97f8-ec2a7bae4c0e.png)

**Comment by theory** [comment]
Those are designed to show you how to use it, not as a complete example of a command. What is it you're trying to do?

**Comment by bwlove319** [comment]
For me (and others?), I tend to copy example commands then edit them to fit my needs.  I'm guessing a lot of people do that.  So it could alleviate some confusion, I think, to put working commands in as examples.

When docs say `this is how it works, and here's a working example`, I generally assume those examples are accurate.
